tests/integration/linux_bridge_test.py

```diff
@@ -189,7 +211,26 @@ def test_add_port_to_existing_bridge(bridge0_with_port0, port1_up):
     assertlib.assert_state(desired_state)
 
 
-def test_linux_bridge_uses_the_port_mac(port0_up, bridge0_with_port0):
+@pytest.mark.xfail(
+    is_fedora(),
+    reason='Fedora 31+ does not use kernel MAC for bridge',
```

Comment:
How about this reason? `On Fedora 31+, users need to explicitly configure the port MAC due to changes to the default systemd config.`.